src/fonts/generate_fonts.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python2
+
+import sys
+import os
+import json
+
+from fontTools.ttLib import TTFont, sfnt
+from fontTools.misc.timeTools import timestampNow
+sfnt.USE_ZOPFLI = True
+
+if len(sys.argv) < 2:
+    print "Usage: %s <font file>" % sys.argv[0]
+    sys.exit(1)
+
+font_file = sys.argv[1]
+font_name = os.path.splitext(os.path.basename(font_file))[0]
+
+# now or SOURCE_DATE_EPOCH, if present
+timestamp = timestampNow()
+
+font = TTFont(font_file, recalcBBoxes=False, recalcTimestamp=False)
+
+font['head'].created = timestamp
+font['head'].modified = timestamp
+
+# remove fontforge timestamps
+if 'FFTM' in font:
+    del font['FFTM']
+
+# remove redundant GDEF table
+if 'GDEF' in font:
+    del font['GDEF']
+
+# remove Macintosh table
+# https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6cmap.html
+font['name'].names = [record for record in font['name'].names if record.platformID != 1]
+font['cmap'].tables = [table for table in font['cmap'].tables if table.platformID != 1]
+
+font['name'].setName(unicode('Version ' + str(timestamp)), 5, 3, 1, 1033)
```

Comment:
Use the timestamp as the version

Reply:
Which is based on when the files in `src/fonts` were last updated.

Reply:
Yes, set via the environment variable `SOURCE_DATE_EPOCH`.